CHANGELOG.md

```diff
@@ -68,6 +68,7 @@
 - `signalfxexporter`: Fix bug to enable timeouts for correlating traces and metrics (#9101)
 - `windowsperfcountersreceiver`: fix exported values being integers instead of doubles (#9138)
 - `prometheusreceiver`: Fix issues with relabelling the `job` and `instance` labels. (#8780)
+- `fluentforwardreceiver`: Fluentforward receiver not releasing port for reloading of OTEL collector (#9111)
```

Comment:
Sorry for one more change about this, but we just had a release and this has to be moved to the unreleased section. I tried updating this PR, but I couldn't push my changes.

Reply:
I have update this push to unreleased section.